src/resources/employees.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+
+const router = express.Router();
+const fs = require('fs');
+const employees = require('../data/employees.json');
+
+module.exports = router;
+
+router.post('/createEmployee', (req, res) => {
```

Comment:
The full route to this endpoint is `/employees/createEmployee`, this seems redundant and it would be better to call this endpoint `/employees/create`. 
```suggestion
router.post('/create', (req, res) => {
```